qiskit/__init__.py

```diff
@@ -18,8 +18,15 @@
 """Main Qiskit public functionality."""
 
 import pkgutil
+import sys
 import warnings
 
+if sys.version_info[0] == 3 and sys.version_info[1] == 5:
+    warnings.warn(
+        "Using Qiskit with Python 3.5 is deprecated as of the 0.11.0 release. "
+        "Support for running Qiskit with Python 3.5 will be removed at the "
+        "Python 3.5 EoL on 10/13/2019.", DeprecationWarning)
```

Comment:
2020 maybe

Reply:
oops sry, too many dates flying around my head this morning. One sec I'll fix it. (I got it right in the release notes at least)